src/shared/constants/status.js

```diff
@@ -133,15 +133,40 @@ export const BRANCH = [
     value: 'airguard',
     label: 'Air National Guard'
   },
+  {
+    value: 'other',
+    label: 'Just want to help out!'
+  }
 ];
 
 export const BRANCH_PROMPT = {
```

Comment:
Since we're hardcoding in the string value of "Branch / Affiliation" as the FormSelect prompt now, regardless of whether the user is active duty, veteran, milspouse, or a supporter who just wants to help out, there's some more cleanup that should be done:
- [ ] Let's go ahead and delete this exported const `BRANCH_PROMPT` from status.js, it's no longer being used
- [ ] Remove the import of the `BRANCH_PROMPT`constant in militaryinfo.js
- [ ] Remove the `onChange` handler within the MilitaryInfo component triggered from the FormSelect
- [ ] Remove the initialization of `this.state` in the constructor of the same component.

Reply:
@jjhampton Hey John, I made the changes you requested. I checked with David on the wording change, so we're good go there. Should I make another PR?

Reply:
@rgoc123 No need to make a new PR - just pushing those changes to this open PR works better. You can continually push changes to a PR while it's still open, which allows you to respond to feedback from reviewers without opening a new one.

I'll take a look at the changes shortly, thanks!